browsers/chrome/_locales/en/messages.json

```diff
@@ -1 +1 @@
-{"appName":{"message":"DuckDuckGo Privacy Essentials","description":"The title of the extension, displayed in the web store"},"appDesc":{"message":"Privacy, simplified. Protect your data as you search and browse. ***Permissions are to block hidden trackers lurking everywhere.***","description":"The short description of the extension, displayed in the web store"}}
```

Comment:
This extension can be installed from Chrome Web Store by Brave and Vivaldi too. Should we try to avoid saying "Chrome"?